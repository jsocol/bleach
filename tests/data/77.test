<div id="5"><script>crypto.generateCRMFRequest('CN=0',0,0,null,'alert(5)',384,null,'rsa-dual-use')</script>//["'`-->]]>]</div>
