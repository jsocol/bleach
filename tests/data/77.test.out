<div id="5">//["'`--&gt;]]&gt;]</div>
